### ISSUE TITLE
Can not update desktop entry for app

### ISSUE THREAD

**Issue by aPollO2k** [karakun/OpenWebStart#388]
I am facing this issue when i run an webstart app

```
java.io.FileNotFoundException: Could not find the javaws binary to create desktop shortcut
	at net.sourceforge.jnlp.util.WindowsDesktopEntry.getJavaWsBin(WindowsDesktopEntry.java:109)
	at net.sourceforge.jnlp.util.WindowsDesktopEntry.createShortcutOnWindowsDesktop(WindowsDesktopEntry.java:83)
	at com.openwebstart.os.win.WinEntryFactory.createDesktopEntry(WinEntryFactory.java:24)
	at com.openwebstart.os.win.WinEntryFactory.updateDesktopEntry(WinEntryFactory.java:13)
	at com.openwebstart.os.MenuAndDesktopEntryHandler.lambda$updateEntries$1(MenuAndDesktopEntryHandler.java:81)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

![2021-02-04 12_48_44-Fehler](https://user-images.githubusercontent.com/5796295/106888814-61401400-66e7-11eb-97ed-0496ed73b759.png)

I'm running OpenWebstart 1.3.2 with 1.8.0.252 Zulu x64 on

- Windows 10 1909 (Build 18363.1316)
- Oracle Java 8 Update 241 installed

Can anyone confirm this?

**Comment by sclassen** [comment]
Could this be the same issue as #243 ?

**Comment by macgibbon** [comment]
I am seeing the same error message with OpenWebstart 1.3.2 but not version 1.3.1.

Looks like a bug from here.

**Comment by sclassen** [comment]
@macgibbon can you reproduce the difference in the behavior between 1.3.1 and 1.3.2
Because 1.3.2 has only a few changes and non of them should be related to the shortcut or filesystem handling.
But if you are certain that there is a change in the behavior between these versions it would be very helpful if you can attach logs from both version such that we can try to find the difference

**Comment by aPollO2k** [comment]
> 
> 
> Could this be the same issue as #243 ?

My Desktop directory is the default path in C:\Users\user\Desktop so i didn't think it is the same issue.

**Comment by sclassen** [comment]
OK, thanks for the info

**Comment by aPollO2k** [comment]
> 
> 
> I am seeing the same error message with OpenWebstart 1.3.2 but not version 1.3.1.
> 
> Looks like a bug from here.

I also can confirm that the message not appear in 1.3.1. After i downgraded to 1.3.1 the desctop entry was created successfully. How can i help to debug this?

**Comment by AndreasEhret** [comment]
can reproduce with 1.3.2 when create menu entry.

![OWS_createMenuEntryProblem](https://user-images.githubusercontent.com/9326167/107747061-25c0cd80-6d17-11eb-9f0e-e1b4b0e6b16c.png)

**Comment by macgibbon** [comment]
I can replicate the issue on this URL

[jnlp:http://tools.transonic.com/RubiksWar/Rubiks.jnlp](jnlp:http://tools.transonic.com/RubiksWar/Rubiks.jnlp)

This is one of Gluon's most excellent samples!

**Comment by sclassen** [comment]
fixed in #393
